### ISSUE TITLE
Add Alt tag to images and REM fonts for WCAG 2.0 Level AA accessibility requirements

### ISSUE THREAD

**Issue by andrew-lof** [ThemeFuse/Brizy#173]
Hi, it would be great for images that get added to also output the ALT text stored in Wordpress Media library for that image. This is needed to make Brizy WCAG 2.0 compliant. 

https://www.w3.org/TR/WCAG20-TECHS/H37.html

Also, if we could set font-size as the relative unit REM instead of only Pixels so it allows for a better text zooming experience which is another WCAG 2.0 recommendation.

https://usability.yale.edu/web-accessibility/articles/zoom-resizing-text

Thanks

**Comment by tinsterman** [comment]
ALT tags are super important and should be linked to the media library.
Also why are the images not being pulled from the media library? They are being added to a seperate breezy folder. This could cause an issue if we want to offload images to a CDN like Amazon S3

**Comment by bagrinsergiu** [comment]
Hi 

Added to Brizy Roadmap: https://trello.com/c/AVcWpZXo/85-yoast-seo-brizy-alt-attributes

Thanks for suggestion.

I’m closing this issue to keep the issue database slim. Feel free to re-open if I’ve misunderstood you.

Sergiu
